bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -23,10 +27,30 @@
   flex: 0;
 }
 
-.center {
+.center,
+.nonVoicePresenter,
+.voicePresenter,
+.nonVoiceUser,
+.voiceUser {
   flex: 1;
 }
 
+.nonVoicePresenter {
+  margin-left: -11.6em;
```

Comment:
You should center things using flex-box properties and not margin. Things should be centered not mattering its size or the parent size. Margin should be used to add margin and not to center things, there is better alternatives to it.